crates/redeem/src/types.rs

```diff
@@ -13,6 +13,8 @@ pub enum Version {
     V1,
     /// RedeemRequestStatus, removed amount_dot and amount_polka_btc
     V2,
+    /// ActiveBlockNumber, btc_height
+    V3,
```

Comment:
Have we already upgraded to V2?

Reply:
I believe that was our last upgrade, correct @sander2?

Reply:
> I believe that was our last upgrade, correct @sander2?

Yes indeed